src/lib-components/BlockStaffArticleList.vue

```diff
@@ -76,18 +98,48 @@ const parsedTextAll = computed(() => {
     ? removeHtmlTruncate(props.description, 250)
     : ''
 })
+
+// Display date based on which data is provided
+const parsedDateDisplay = computed(() => {
+  if (props.ongoing === true)
+    return 'Ongoing'
+  else if (props.endDate)
```

Comment:
this is not clear to me
 else if (props.endDate)
    return formatSeriesDates(props.startDate, props.endDate, 'shortWithYear')
  else if (props.startDate)
    return formatSeriesDates(props.startDate, props.startDate)
    
    why are we checking in separately if condition startdate and endate exists and formatting it differently?

Reply:
also should this date display be specific for ftva theme and ftvaeventseries section handle

Reply:
I wasn't sure if we wanted to be that specific in the component